amgut/lib/data_access/redcap.py

```diff
@@ -0,0 +1,115 @@
+from urllib import urlencode
+from json import loads
+
+from tornado.web import HTTPError
+from tornado.httpclient import HTTPClient
+
+from amgut.lib.config_manager import AMGUT_CONFIG
+from amgut.lib.data_access.sql_connection import TRN
+
+
+def _make_request(data):
+    body = urlencode(data)
+    client = HTTPClient()
+    response = client.fetch(AMGUT_CONFIG.redcap_url, method='POST',
+                            headers=None, body=body)
+    # Response not always JSON, so can't always use loads
+    if '{"error"' in response.body:
+        raise HTTPError(400, loads(response.body)['error'])
+    return response.body
+
+
+def get_survey_url(record, instrument='ag_human_en_us'):
+    """Genereates a new survey link for a given record
+
+    Parameters
+    ----------
+    record : int
+        Redcap record (consent) to attach this survey to
+    instrument : str, optional
+        The instrument (survey) to get the URL for. Default ag_human_en_us
+
+    Returns
+    -------
+    str
+        URL for the survey
+    """
+    with TRN:
```

Comment:
Ah, okay. Would it be possible to add documentation either to the method, or perhaps at the top of the file which describes how to interpret the IDs? I worry that this could easily be confusing again 6 months from now